### PR TITLE
Fixed a data type error caused due to prometrix library

### PR DESCRIPTION
**Pull request by ganeshrvel**

### CODE REVIEW

File: src/robusta/core/playbooks/prometheus_enrichment_utils.py [hunk 1]
```diff
@@ -63,7 +63,7 @@ def __prepare_promql_query(provided_labels: Dict[Any, Any], promql_query_templat
     return promql_query
 
 
-def custom_query_range(
+def safe_custom_query_range(
```
Comment (arikalon1) [inline src/robusta/core/playbooks/prometheus_enrichment_utils.py:66]: Can we remove this function?
it's confusing (has the same name of the one from prometrix), and is used only once in line 102
looks redundant

**Review by arikalon1 — APPROVED** [review]
nice work
left one commment

File: src/robusta/core/playbooks/prometheus_enrichment_utils.py (hunk 1 above)
Comment (ganeshrvel) [reply src/robusta/core/playbooks/prometheus_enrichment_utils.py]: done

**Review by arikalon1 — APPROVED** [review]
nice work

**Pull request merged**